src/Microsoft.Health.Fhir.Core/Features/Search/SearchOptions.cs

```diff
@@ -26,6 +26,28 @@ internal SearchOptions()
         {
         }
 
+        internal SearchOptions(SearchOptions other)
```

Comment:
Where is this used?

Reply:
Added this to use when creating a new `SqlSearchOptions` object using an existing `SearchOptions` object.